tests/src/whisk/consul/ConsulKVTests.scala

```diff
@@ -35,19 +36,32 @@ class ConsulKVTests extends FlatSpec with Matchers {
     private val config = new WhiskConfig(requiredProperties)
     private val consul = new ConsulKV(config.consulServer)
 
+    /*
```

Comment:
The retry method looks like something general that we could have in a utility and reuse elsewhere in tests. No?